.travis.yml

```diff
@@ -2,10 +2,10 @@ sudo: false
 
 language: scala
 scala:
-  - 2.11.12
   - 2.12.8
+  - 2.13.0
 jdk:
-  - oraclejdk8
+  - oraclejdk9
```

Comment:
Did you change this to oraclejdk9 because Travis complained about it or because you want it specifically? Most people I know in the Scala world still use openjdk8 (not oraclejdk8, which is not supported by Travis), so I'd prefer if we kept a build with it.

Reply:
Travis complained: https://travis-ci.org/ruippeixotog/scala-scraper/jobs/573826740#L166

If you know any workaround, please let me know :)

Reply:
Yes, changing it from `oraclejdk8` to `openjdk8` should do the trick :)

Reply:
Fixed!